### DISCUSSION TITLE
Import / Export maps to shared storage

### DISCUSSION THREAD

**Discussion by blockx** [organicmaps/organicmaps#7923] [Ideas]
Hi,

For people like myself who have multiple installations of Organic Maps it would be great to have the facility to import and export maps to/from shared storage so a single download of a map or collection of maps could be used by several installations on different devices.

Thanks,

**Comment by biodranik** [comment]
You can likely use adb pull/adb push if maps are stored on external storage/SD-card (check OM settings).

**Reply by blockx** [reply]
Hmm.  I think using adb is beyond the capabilities or inclinations of the average user though ?  Also, I don't want to set developer access to all my devices so that would not be a solution for me.

But many thanks for your suggestion,

**Reply by biodranik** [reply]
Please follow https://github.com/organicmaps/organicmaps/issues/1801
